frontend/templates/troubleshooting/index.tsx

```diff
@@ -95,125 +124,153 @@ const Wiki: NextPageWithLayout<{
+                              <ModalHeader>{title}</ModalHeader>
+                           </VisuallyHidden>
+                           <ModalCloseButton />
+                           <VisuallyHidden></VisuallyHidden>
+                           <ModalBody padding={0}>
+                              <Image
+                                 src={mainImageUrlLarge}
+                                 width="100%"
+                                 height="auto"
+                                 alt={title}
+                              />
+                           </ModalBody>
+                        </ModalContent>
+                     </Modal>
+                     <VStack alignItems="flex-start" spacing={2}>
+                        <HeadingSelfLink
+                           as="h1"
+                           fontSize="3xl"
+                           fontWeight="medium"
+                           selfLinked
+                           id="top"
+                           mt={0}
+                        >
+                           {wikiData.title}
+                        </HeadingSelfLink>
+                        <AuthorInformation
+                           lastUpdatedDate={lastUpdatedDate}
+                           authors={wikiData.authors}
+                           historyUrl={wikiData.historyUrl}
+                        />
+                     </VStack>
+                  </HStack>
+                  {wikiData.introduction.map((intro) => (
+                     <IntroductionSection key={intro.heading} intro={intro} />
+                  ))}
+                  {wikiData.solutions.length > 0 && (
+                     <Box
+                        borderTop="1px"
+                        borderColor="gray.300"
+                        mt={{ base: 4, sm: 6 }}
+                     >
+                        <HeadingSelfLink
+                           as="h2"
+                           fontSize="20px"
+                           fontWeight="semibold"
+                           selfLinked
+                           id="causes"
+                        >
+                           {'Causes'}
+                        </HeadingSelfLink>
+                        <TableOfContents
+                           solutions={wikiData.solutions}
+                           problems={wikiData.linkedProblems}
+                        />
+                        <Stack spacing={3} mt={{ base: 7, sm: 10 }}>
+                           {wikiData.solutions.map((solution, index) => (
+                              <SectionCard
+                                 key={solution.heading}
+                                 index={index + 1}
+                                 solution={solution}
+                              />
+                           ))}
+                        </Stack>
+                     </Box>
+                  )}
+                  <Conclusion conclusion={filteredConclusions} />
```

Comment:
Will this change drop the `Related Pages` from the live version? Is that okay?

Reply:
Yes, this is already a Business Rule, I just moved it up the stack one component.